connectors/hive/README.md

```diff
@@ -1,5 +1,5 @@
 # Hive Connector
-This project is a library to make Hive read Delta tables. The project provides a uber JAR `delta-hive-assembly_<scala_version>-<delta_connectors_version>.jar` to use in Hive. You can use either Scala 2.11, 2.12 or 2.13. The released JARs are available in the [releases](https://github.com/delta-io/connectors/releases) page. Please download the uber JAR for the corresponding Scala version you would like to use.
+This project is a library to make Hive read Delta tables. The project provides a uber JAR `delta-hive-assembly_<scala_version>-<delta_connectors_version>.jar` to use in Hive. You can use either Scala 2.11, 2.12 or 2.13. The released JARs are available in the [releases](https://github.com/delta-io/delta/releases) page. Please download the uber JAR for the corresponding Scala version you would like to use.

```

Comment:
Does this need updated to point to the pre-3.0 release?

Reply:
Done